static/js/containers/admin/ChannelModerationPage.js

```diff
@@ -58,11 +58,16 @@ export class ChannelModerationPage extends React.Component<Props> {
   }
 
   loadData = async () => {
-    const { dispatch, channelName } = this.props
+    const { dispatch, channelName, loaded } = this.props
 
     try {
-      await dispatch(actions.channels.get(channelName))
-      await dispatch(actions.reports.get(channelName))
+      if (!loaded) {
```

Comment:
I think `loaded` may not be getting set properly.  If I report some comments (in same browser, or in different browser under different account), then navigate to the 'Reported Content' tab, the reported comments do not appear until I refresh the page.